srtcore/queue.h

```diff
@@ -90,39 +90,33 @@ class CUnitQueue
     ~CUnitQueue();
 
 public: // Storage size operations
-        /// Initialize the unit queue.
-        /// @param [in] size queue size
-        /// @param [in] mss maximum segment size
-        /// @param [in] version IP version
-        /// @return 0: success, -1: failure.
-    int init(int size, int mss, int version);
-
-    /// Increase (double) the unit queue size.
-    /// @return 0: success, -1: failure.
-
-    int increase();
-
-    /// Decrease (halve) the unit queue size.
+    /// Initialize the unit queue.
+    /// @param [in] size queue size
+    /// @param [in] mss maximum segment size
+    /// @param [in] version IP version
     /// @return 0: success, -1: failure.
-
-    int shrink();
+    int init(int size, int mss, int version);
 
 public:
     int size() const { return m_iSize - m_iCount; }
     int capacity() const { return m_iSize; }
 
 public: // Operations on units
-        /// find an available unit for incoming packet.
-        /// @return Pointer to the available unit, NULL if not found.
+    /// find an available unit for incoming packet.
+    /// @return Pointer to the available unit, NULL if not found.
     CUnit* getNextAvailUnit();
 
     void makeUnitFree(CUnit* unit);
 
     void makeUnitGood(CUnit* unit);
 
-public:
     inline int getIPversion() const { return m_iIPversion; }
 
+private:
+    /// Increase (double) the unit queue size.
```

Comment:
```suggestion
    /// Increase the unit queue size by initial size.
```
It does not always double the whole size, right?

Reply:
Right. Every time it just adds the same amount (32 units previously, 256 with this PR).
Might make sense to improve this by allocating 1,62*current_size. 🤔